frontend/src/components/details-car-panel/details-car-panel.tsx

```diff
@@ -29,20 +32,21 @@ const DetailsCarPanel: FC<DetailsCarPanelPropsType> = ({
   const [deleteWishlist] = useDeleteWishlistMutation();
 
   const handleCreateWishlist = async (): Promise<void> => {
-    const data: WishlistInput = complectationId
-      ? { modelId }
-      : { complectationId };
-
+    const data: WishlistInput = modelId ? { modelId } : { complectationId };
     await createWishlist(data);
   };
 
   const handleDeleteWishlist = async (): Promise<void> => {
-    const data: WishlistInput = complectationId
-      ? { modelId }
-      : { complectationId };
+    const data: WishlistInput = modelId ? { modelId } : { complectationId };
     await deleteWishlist(data);
   };
```

Comment:
@stanislav-ostr It would be better to use `WishlistContext` here like you did in PR #237